week-4/Homework/mandatory/2-writers.js

```diff
@@ -39,6 +39,19 @@ let writers = [
   }
 ];
 
+// let hasElement = (alive, writer) => writer.alive === alive;
+
+// let writerIsAlive = writers.filter(x => hasElement(true, x));
+
+// console.log(writerIsAlive);
+
+// writers.forEach(element => console.log(`Hi, my name is ${element.firstName} ${element.lastName}. I am ${element.age} years old, and work as a ${element.occupation}.`))
+
+// writers.forEach(element => element.alive ? console.log(`Hi, my name is ${element.firstName} ${element.lastName}.

```

Comment:
👏This works! Try to use the `aliveWriters` array you created before.
Why not even chain `.filter` and `.forEach` methods?